.gitignore

```diff
@@ -224,3 +224,6 @@ licenses/LICENSES-ui.txt
 
 # Packaged Breeze2 on Windows
 /Breeze2.exe
+
+# Generated pre_commit_id files
+/dev/breeze/src/airflow_breeze/pre_commit_ids.py
```

Comment:
Thiis shared from the other commit I guess :) ?

Reply:
Yeah, I will fix it up :)